libraries/DataFlash/examples/DataFlash_AllTypes/DataFlash_AllTypes.cpp

```diff
@@ -91,7 +91,7 @@ class DataFlashTest_AllTypes : public AP_HAL::HAL::Callbacks {
 private:
 
     AP_Int32 log_bitmask;
-    DataFlash_Class dataflash{"DF AllTypes 0.2", log_bitmask};
+    DataFlash_Class dataflash{log_bitmask};
```

Comment:
You have to add an AP_FWVersion to all examples that have a dataflash, otherwise they'll crash.